docs/adminInvitation/admin-invitation.yaml

```diff
@@ -0,0 +1,85 @@
+paths:
+  /admin-invitations:
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - Admin invitations
+      summary: Invite admins
+      description: Creates and sends admin invitations.
+      produces:
+        - application/json
+      parameters:
+      - in: header
+        name: Accept-Language
+        type: string
+        enum: [en, ua]
+        required: true
+      requestBody:
+        required: true
+        description: Provide required data to invite admins.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/invite-admins'
+            example:
+              emails:
+              - johndoe@gmail.com
+              - vasya@gmail.com
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/admin-invitations'
+              example:
+                - _id: 6255bc080a75adf9223df100
+                  email: johndoe@gmail.com
+                  dateOfInvitation: 2022-09-02T11:59:53.243Z
+                - _id: 6255bc080a75adf9223df101
+                  email: vasya@gmail.com
+                  dateOfInvitation: 2022-09-02T11:59:53.243Z     
+        401:
```

Comment:
shouldn't it have 403 forbidden error as well?

Reply:
ok